pkg/common/constants/constants.go

```diff
@@ -23,8 +23,14 @@ const (
 	DefaultReplicationFactor = 1
 
 	// The name of the configmap used to hold eventing-kafka settings
-	SettingsConfigMapName          = "config-kafka"
+	SettingsConfigMapName = "config-kafka"
+
+	// Mount path of the configmap used to hold eventing-kafka settings
+	SettingsConfigMapMountPath = "/etc/config-kafka"
```

Comment:
should we leverage here the name of the `SettingsConfigMapName` ? 

well, could be too much :smile:

Reply:
yeah, thanks!